data/initrd/scripts/remote_log_setup

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+#Requires the LOGHOST variable to be set
```

Comment:
Again, that's all not necessary. You don't really modify rsyslog.conf - it's always the same file! So just create the config you like and include the file directly in initrd.file_list replacing the default config.